.github/workflows/ci.yml

```diff
@@ -400,6 +400,9 @@ jobs:
             - name: Build & Test (with entitlements)
               run: yarn csb:turbo-no-sdk
 
+            - name: Run River Tests Against Legacy Spaces (with entitlements)
+              run: yarn workspace @river-build/sdk run test:ci:multi:ent
```

Comment:
@clemire did you mean to use `test:ci:multi:ent:legacy` here?

Reply:
I did - this is a new suite that uses the legacy space creation endpoint on the space diamond and runs all of our entitlement tests. The existing line now runs against the new space creation endpoint. This test suite validates backwards compatibility with V1 spaces.

Reply:
I think this could significantly increase how long it take the multinode_ent to run though so I'm willing / happy to split this out into a different job for engineer workflow reasons if you're concerned

Reply:
@clemire yes but you're not using it :) you're using `test:ci:multi:ent:legacy`

Reply:
ahhhhh ok thanks for noticing!

Reply:
...I put it in a separate job @mechanical-turk because it was taking > 30min to finish